src/MusicTheory/Pitch.elm

```diff
@@ -0,0 +1,13 @@
+module MusicTheory.Pitch exposing (Pitch(..))
+
+import MusicTheory.Internal.PitchClass as PitchClass exposing (Offset, PitchClass)
+import MusicTheory.Octave as Octave exposing (Octave)
+
+
+type Pitch
+    = Pitch PitchClass Octave
+
+
+pitch : PitchClass -> Octave -> Pitch
```

Comment:
I agree that `PitchClass` to `Pitch` conversion should be an option. But I don't think it should be the same as a `pitch` constructor.

If I want to make a `Pitch`, it would be a nicer interface to just pass `Letter -> Accidental -> Octave`. Otherwise I have to make a `PitchClass` that I might not care about, just to make a `Pitch`.

So I would suggest we make:
`pitch : Letter -> Accidental -> Octave -> Pitch`
and
`fromPitchClass : PitchClass -> Octave -> Pitch`